src/Playlist.java

```diff
@@ -0,0 +1,22 @@
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.TimeZone;
+
+public class Playlist {
+
+    public int playlistLength(Song... songs ){
+        int playingTime = 0;
+        for (Song song : songs) {
+            playingTime += song.getDuration();
+        }
+        return playingTime;
+    }
+
+    public void printPlaylistDuration(int durationSong){
```

Comment:
Why are you complicating things so much? We just need to turn seconds into actual time. Did you google how to do it? Hint -> https://docs.oracle.com/javase/8/docs/api/java/time/Duration.html

Reply:
I did not find a method here that would convert seconds to HH: MM: SS format.

Reply:
I'd have to create an additional method that would replace any part of the Duration object with a String.

Reply:
There is a way to do it using this class. Look closer.